src/pages/Apps/Apps.scss

```diff
@@ -42,9 +43,14 @@
       }
 
       .app-name {
-        margin-top: 10px;
+        margin-top: 6px;
         text-align: center;
       }
     }
   }
+
+  .close-btn-boundaries{
```

Comment:
Missed space. This declaration can be inside `.app-shortcut`.

Reply:
I added the space. However there is no point in replicating the HTML-Structure in your styling. It just adds unnecessary specificity too the CSS selector. And you couple your CSS and HTML tightly.